data/json/items/armor/suits_protection.json

```diff
@@ -844,6 +844,18 @@
     "looks_like": "touring_suit",
     "color": "dark_gray",
     "covers": [ "LEGS", "TORSO", "ARMS" ],
+    "coverage_data": [
```

Comment:
I think a better name would be `armor_data`, not `coverage_data`.

Reply:
armor_data is already used elsewhere, it was my first choice too. Any other suggestions?

Reply:
Ah, good point. Potentially the less verbose `armor`? I'm not sure, someone else may have something.